packs/boundary_worker/README.md

```diff
@@ -0,0 +1,126 @@
+# Nomad Pack: (HCP) Boundary Worker
+
+![Nomad Pack: Boundary Worker](https://assets.workloads.io/nomad-pack-registry/boundary_worker.png)
+
+## Table of Contents
+
+<!-- TOC -->
+* [Nomad Pack: (HCP) Boundary Worker](#nomad-pack-hcp-boundary-worker)
+  * [Table of Contents](#table-of-contents)
+  * [Requirements](#requirements)
+  * [Usage](#usage)
+    * [Application](#application)
+    * [Nomad](#nomad)
+    * [Outputs](#outputs)
+  * [Notes](#notes)
+    * [Resource Sizing](#resource-sizing)
+  * [Author Information](#author-information)
+  * [License](#license)
+<!-- TOC -->
+
+## Requirements
+
+- HashiCorp Nomad `1.7.x` or [newer](https://developer.hashicorp.com/nomad/install)
+- HashiCorp Nomad Pack `0.1.0` or [newer](https://releases.hashicorp.com/nomad-pack/)
+- Nomad Task Driver(s) for [`raw_exec`](https://developer.hashicorp.com/nomad/docs/drivers/raw_exec)
+- HCP Boundary Cluster ID, see [Notes](#notes)
+
+## Usage
+
+The `boundary_worker` Pack can be run with a local fileset (commonly used when developing the Pack), or via the [@workloads Nomad Pack Registry](https://github.com/workloads/nomad-pack-registry).
+
+A Pack available _locally_ may be run like so:
+
+```shell
+nomad-pack run ./packs/boundary_worker
+```
+
+A Pack available via the [@workloads Nomad Pack Registry](https://github.com/workloads/nomad-pack-registry) may be run like so:
+
+```shell
+# add the Pack Registry
+nomad-pack registry add workloads github.com/workloads/nomad-pack-registry
+
+# run the Pack
+nomad-pack run boundary_worker --registry=workloads
+```
+
+<!-- BEGIN_PACK_DOCS -->
+
+### Application
+
+This section describes Application-specific configuration.
+
+| Name                                     | Description                                                                                                        | Default |
+| ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------ | ------- |
+| app_boundary_helper_output_file_mode     | File Mode of the Output File created by the Boundary Helper binary.                                                | `"0644"` |
+| app_boundary_helper_path                 | Path to the Boundary Helper binary.                                                                                | `"boundary-helper"` |
+| app_cors_allowed_origins                 | Allowed CORS Origins for the Boundary Worker.                                                                      | `["*"]` |
+| app_cors_enabled                         | Toggle to enable CORS support for the Boundary Worker.                                                             | `true` |
+| app_disable_mlock                        | Toggle to disable MLock for the Boundary Worker.                                                                   | `true` |
```

Comment:
Same as above, what's the impact? (and I would presume that mlocking wouldn't work with the `exec` task driver without some extra capabilities)

Reply:
And why is it true by default?

Reply:
A remnant from building it for non-HCP and speed of development. Updated and removed! ❤️

Reply:
I develop this on macOS, where `mlock` triggers a warning, as `boundary` isn't running inside a container, but on the host itself.

I've updated the docs to be more clear - thank you for the spot!